packages/react-i18n/src/constants.ts

```diff
@@ -1,11 +1,19 @@
 export enum DateStyle {
+  ExtraLong = 'ExtraLong',
```

Comment:
Should that be called `LongWithTime` to indicate it shows the time as well?

Reply:
Ya, I feel like the name could be better.

Reply:
Yea i would vote for `LongWithTime` as well

Reply:
OK, I've gone with `LongWithTime` for now, though it bothers me that `Long` includes the weekday and `LongWithTime` does not.

Reply:
It bothers me as well since it doesn't exactly follow the polaris guidelines. Maybe we should remove this and leave it up to the consumer to provide custom formatting